fire_atlas_editor/lib/widgets/image_selection_container.dart

```diff
@@ -25,7 +25,7 @@ class ImageSelectionContainer extends StatelessWidget {
   });
 
   @override
-  Widget build(_) {
+  Widget build(context) {
```

Comment:
parameter type

Reply:
Why? Dart infers it for you, especially in a well know method like `Widget#build` I don't think we need to put the type.

Reply:
This isn't a public API, this is totally internal.

If this was for an api of course we have to put the type, otherwise it would be dynamic...